src/main/java/org/spongepowered/common/mixin/core/tileentity/MixinTileEntityJukebox.java

```diff
@@ -49,12 +49,18 @@
 
     @Override
     public void playRecord() {
-        if (getRecord() != null) {
-            this.world.playEvent(null, 1005, this.pos, Item.getIdFromItem(getRecord().getItem()));
+        if (!getRecord().isEmpty()) {
+            this.world.playEvent(null, 1010, this.pos, Item.getIdFromItem(getRecord().getItem()));
```

Comment:
Looks like a magic number that you use in more than a single location. Maybe we could move it to a constant.